plasma_framework/test/src/framework/ExitGameController.test.js

```diff
@@ -20,7 +20,15 @@ contract('ExitGameController', () => {
         this.dummyTxType = 1;
         this.controller.registerExitGame(this.dummyTxType, this.dummyExitGame.address);
 
-        this.dummyToken = constants.ZERO_ADDRESS;
+        // take any random contract address as token
+        this.dummyToken = (await DummyExitGame.new()).address;
+    });
+
+    describe('constructor', () => {
+        it('inits the queue for ETH', async () => {
```

Comment:
Standard practice when using Mocha is to start the test description with 'should...' e.g.
`it('should init the queue for ETH'`
It just reads better, but it's not a big deal if you prefer not to do that.

Reply:
thanks, didn't know about that!